### PR TITLE
Standardize file names

### PR DESCRIPTION
**Pull request by mjholub**
*PR to correct base.*

It's a bit odd for ayu themes to be the only ones with names not joined by an underscore. Someone may then wonder why their theme setting isn't working if the names are not standardized.

### CODE REVIEW

File: lua/base46/themes/gruvbox.lua [hunk 1]
```diff
@@ -52,7 +55,27 @@ M.base_16 = {
   base0F = "#d65d0e",
 }
 
+<<<<<<< HEAD
```
Comment (154pinkchairs) [inline lua/base46/themes/gruvbox.lua:58]: I rebased onto maintainer's dev branch, I don't know why wasn't that correctly accounted for.

Comment (mjholub) [inline lua/base46/themes/gruvbox.lua]: errant rebase?

File: lua/base46/themes/gruvbox.lua [hunk 2]
```diff
@@ -52,7 +55,23 @@ M.base_16 = {
   base0F = "#d65d0e",
 }
 
-M.type = "dark"
```
Comment (mjholub) [inline lua/base46/themes/gruvbox.lua:55]: idk why it shows as a change even though this is how this file looks like on dev branch

**Comment by siduck** [comment]
@154pinkchairs fork dev branch and then try adding your changes

**Pull request closed without merging**